diagnostics/diag.go

```diff
@@ -98,7 +98,13 @@ func (di *DiagInfo) Marshal() []byte {
 }
 
 func (d *Diagnostics) getPeers() []peer.ID {
-	return d.host.Network().Peers()
```

Comment:
yeah, that would solve the issue. Im just wondering what the use of `Peers()` is if not this?